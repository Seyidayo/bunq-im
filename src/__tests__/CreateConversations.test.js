import renderer from 'react-test-renderer';
import CreateConversations from 'app/components/UI/molecules/CreateConversationsModal';
import { BrowserRouter } from 'react-router-dom';

describe('<CreateConversations />', () => {
  it('It renders correctly and matches snapshot', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <CreateConversations />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
