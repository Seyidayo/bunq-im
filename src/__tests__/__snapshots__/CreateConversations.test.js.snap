// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateConversations /> It renders correctly and matches snapshot 1`] = `null`;
